paymentsheet-example/src/androidTest/java/com/stripe/android/test/core/PlaygroundTestDriver.kt

```diff
@@ -313,6 +314,54 @@ internal class PlaygroundTestDriver(
         teardown()
     }
 
+    fun savePaymentMethodInCustomerSheet(
+        testParameters: TestParameters,
+        values: FieldPopulator.Values = FieldPopulator.Values(),
+        populateCustomLpmFields: FieldPopulator.() -> Unit = {},
+    ): PlaygroundState? {
+        setup(
+            testParameters.copyPlaygroundSettings { settings ->
+                settings.updateConfigurationData { configurationData ->
+                    configurationData.copy(
+                        integrationType = PlaygroundConfigurationData.IntegrationType.CustomerSheet
+                    )
+                }
+            }
+        )
+
+        launchCustomerSheet()
+
+        if (isManagePaymentMethodScreen()) {
```

Comment:
Should we make the individual tests aware of this instead of doing it automatically? 

I think we did this approach in payment sheet, but it was mostly to preserve the existing tests passing if I remember right.

Reply:
Probably yeah. With these initial cases, we shouldn't end up in this screen since we have a new customer. Otherwise, it is a bug.